bin/letsencrypt_hooks

```diff
@@ -33,13 +33,19 @@ clean_challenge() {
 
 deploy_cert() {
   local DOMAIN="${1}" KEYFILE="${2}" CERTFILE="${3}" FULLCHAINFILE="${4}" CHAINFILE="${5}" TIMESTAMP="${6}"
+  local EXPIRY=$(date --date="$(openssl x509 -enddate -noout -in "$CERTFILE"|cut -d= -f 2)" +%s)
+  if [ $? -ne 0   ]; then
+    echo "failed to get the expiry date."
+    exit 1
```

Comment:
I am not sure whether this should really be a fatal error.

Reply:
I tend to agree as the certificate deployment will work fine even if EXPIRY is not set.
In addition I see that the check would actually never fail. When declaring and setting a local variable in a single command, it's first set and then restricted to local scope, so $? reflects the local declaration, not the variable assignment. 

I propose testing if $EXPIRY is empty and if so logging the problem but skip the fatal error. What do you think?